packages/jest-haste-map/src/__tests__/index.test.js

```diff
@@ -108,18 +108,25 @@ const object = data => Object.assign(Object.create(null), data);
 const createMap = obj => new Map(Object.keys(obj).map(key => [key, obj[key]]));
 
 // Jest toEqual does not match Map instances from different contexts
```

Comment:
@mjesun do you think we should fix this in Jest matchers? Given it's a very unfrequent use case we might just use this solution.